### PR TITLE
removed deprecated popup menu extension point

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/ToggleXtextNatureCommand.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	private static final Logger log = Logger.getLogger(ToggleXtextNatureCommand.class);
+
+	@Override
+	public Object execute(ExecutionEvent event) throws ExecutionException {
+		ISelection selection = HandlerUtil.getCurrentSelection(event);
+		if (selection instanceof IStructuredSelection) {
+			for (Iterator<?> it = ((IStructuredSelection) selection).iterator(); it.hasNext();) {
+				Object element = it.next();
+				IProject project = null;
+				if (element instanceof IProject) {
```
Comment (kthoms) [inline org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/ToggleXtextNatureCommand.java]: IProject extends IAdaptable and should be fine to use the second clause only

Comment (cdietrich) [reply org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/ToggleXtextNatureCommand.java]: did just stupid copy&paste

File: org.eclipse.xtext.ui.shared/plugin.properties [hunk 2]
```diff
@@ -4,6 +4,8 @@ XtextBuilder.name=Xtext Project Builder
 XtextNature.name=Xtext Project Nature
 XtextNature.description=Adds an icon decoration to projects with Xtext enabled.
 XtextNatureAddAction.label=Convert to Xtext Project
+XtextNatureAddCommand.label=Convert to Xtext Project
```
Comment (kthoms) [inline org.eclipse.xtext.ui.shared/plugin.properties]: I know there is a difference between the deprecated action and the new command, but shall we just reuse the message? It's just a "wrong" key name. What do you think?

Comment (cdietrich) [reply org.eclipse.xtext.ui.shared/plugin.properties]: i think i even can remove the old key

**Comment by kthoms** [comment]
Besides the 2 comments OK.

File: org.eclipse.xtext.ui.shared/plugin.properties (hunk 2 above)
Comment (cdietrich) [reply org.eclipse.xtext.ui.shared/plugin.properties]: updated

**Pull request merged**